pyvelox/README.md

```diff
@@ -34,6 +34,12 @@ Run the script referenced [here](https://github.com/facebookincubator/velox#sett
 
 
 #### Install PyVelox
+
+You can install PyVelox from pypi without the need to build it from source as we provide wheels for Linux and macOS (x86_64):
```

Comment:
Maybe we should move the Install PyVelox to after pre-requisites and before From Source